packages/integration-karma/test/component/spellcheck-attribute/index.spec.js

```diff
@@ -0,0 +1,50 @@
+import { createElement } from 'lwc';
+
+import Container from 'x/container';
+
+function getSpellcheckValue(container, selector) {
+    const elm = container.shadowRoot.querySelector(selector);
+
+    return elm.getAttribute('spellcheck');
+}
+
+describe('spellcheck attribute', () => {
+    it('should cast to true static values other than false', () => {
```

Comment:
The term "casting" is traditionally used for signaling to the compiler that a type conversion is safe. 
```suggestion
    it('should render the correct attribute value for static values', () => {
```